docs/configuring-playbook-appservice-draupnir-for-all.md

```diff
@@ -36,7 +34,7 @@ As noted in the Draupnir install instructions the control room is sensitive. **A
 
 Next, set an alias to the management room.
 
-This alias can be anything you want. However, for increased security during the setup phase, it is recommended to make this alias be a random string. When it has been locked down after setup phase, you can give your room a secondary human readable alias.
+This alias can be anything you want.
```

Comment:
Since the suggestion about the alias is no longer relevant, I guess you may simply remove the line.